examples/plot_FNO_darcy.py

```diff
@@ -95,18 +105,61 @@
               training_loss=train_loss,
               eval_losses=eval_losses)
 
+# %%
+# .. plot_preds :
+# Visualizing predictions
+# ------------------------
+# Let's take a look at what our model's predicted outputs look like. 
+# Again note that in this example, we train on a very small resolution for
+# a very small number of epochs.
+# In practice, we would train at a larger resolution, on many more samples.
+
+test_samples = test_loaders[16].dataset
+
+fig = plt.figure(figsize=(7, 7))
+for index in range(3):
+    data = test_samples[index]
+    data = data_processor.preprocess(data, batched=False)
+    # Input x
+    x = data['x']
+    # Ground-truth
+    y = data['y']
+    # Model prediction
+    out = model(x.unsqueeze(0))
+
+    ax = fig.add_subplot(3, 3, index*3 + 1)
+    ax.imshow(x[0], cmap='gray')
+    if index == 0: 
+        ax.set_title('Input x')
+    plt.xticks([], [])
+    plt.yticks([], [])
+
+    ax = fig.add_subplot(3, 3, index*3 + 2)
+    ax.imshow(y.squeeze())
+    if index == 0: 
+        ax.set_title('Ground-truth y')
+    plt.xticks([], [])
+    plt.yticks([], [])
+
+    ax = fig.add_subplot(3, 3, index*3 + 3)
+    ax.imshow(out.squeeze().detach().numpy())
+    if index == 0: 
+        ax.set_title('Model prediction')
+    plt.xticks([], [])
+    plt.yticks([], [])
+
+fig.suptitle('Inputs, ground-truth output and prediction (16x16).', y=0.98)
+plt.tight_layout()
+fig.show()
+
 
 # %%
-# Plot the prediction, and compare with the ground-truth 
-# Note that we trained on a very small resolution for
-# a very small number of epochs
-# In practice, we would train at larger resolution, on many more samples.
-# 
-# However, for practicity, we created a minimal example that
-# i) fits in just a few Mb of memory
-# ii) can be trained quickly on CPU
-#
-# In practice we would train a Neural Operator on one or multiple GPUs
+# .. zero_shot :
+# Zero-shot super-resolution
```

Comment:
Let's make Sure this gets rendered

Reply:
This is rendered properly in the local build